cg/meta/demultiplex/validation.py

```diff
@@ -56,18 +56,20 @@ def validate_samples_have_fastq_files(flow_cell: FlowCellDirectoryData) -> None:
         fastq_files: Optional[List[Path]] = get_sample_fastqs_from_flow_cell(
             flow_cell_directory=flow_cell.path, sample_internal_id=sample_id
         )
-        if not fastq_files:
-            raise MissingFilesError(f"Sample {sample_id} has no fastq files in flow cell")
-    LOG.debug("Flow cell has fastq files for all samples")
+        if fastq_files:
+            LOG.debug(f"Flow cell {flow_cell.id} has at least one sample with fastq files")
+            return
```

Comment:
Is this really a meaningful check? What is the original issue we are trying to resolve?
It seems unlikely that none of the samples on a flow cell would have fastqs

Reply:
The issue Kalle encountered was that the file-structure was not caught by the patterns and no files were found. But reads were added to status_db. As @ChrOertlin mentioned this could be added when parsing sequencing metrics, but I don't really want it there 😬

Reply:
We can leave the per sample check for another PR, as this one checks for structure.

Reply:
Maybe a better way to do this would be to raise an exception in the function which is supposed to return all fastq file paths?